indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -47,4 +48,154 @@
             $('#payment-summary').effect('highlight', 800);
         });
     }
+
+    function setupStaticURLGeneration() {
+        $('.js-static-url').on('click', function(event) {
+            var $this = $(this);
+            $.ajax({
+                method: 'POST',
+                url: $this.data('href'),
+                success: function(result) {
+                    var content = $('.registrations > .clipboard-dialog').clone();
+                    content.find('input:text').val(result.url);
+
+                    $this.qtip({
```

Comment:
Probably not in a resuable way. But a util like `$.fn.showCopyURL(url.` would be nice:

```
$.ajax({
    ...,
    success: function(data) {
        $this.showCopyURL(data.url);
    }
});
```